### ISSUE TITLE
Crash when printing sat witnesses

### ISSUE THREAD

**Issue by rodrigo7491** [usi-verification-and-security/golem#29]
With [CHC-LIA-Lin_110](https://github.com/chc-comp/chc-comp22-benchmarks/blob/main/LIA-Lin/chc-LIA-Lin_110.smt2.gz), Golem (v0.4.0) produces an output when run without arguments, but with `----print-witness`, it crashes with the following message:

```
terminate called after throwing an instance of 'std::out_of_range'
  what():  _Map_base::at
Aborted (core dumped)
```

**Comment by rodrigo7491** [comment]
This was fixed in 2a82a5fa48ea1b6668ae58695ce07f11f7748ee9, right @blishko?

**Comment by blishko** [comment]
Yes!
